src/SIL.XForge.Identity/Controllers/Account/StatusModel.cs

```diff
@@ -0,0 +1,13 @@
+
+
+using System.ComponentModel.DataAnnotations;
+
+
+namespace SIL.XForge.Identity.Controllers.Account
```

Comment:
This class is not used, so you can remove it. I would just revert all code changes on the backend, since the backend doesn't need any changes for this PR.